app/experimenter/experiments/api/v5/types.py

```diff
@@ -405,6 +408,39 @@ def resolve_targeting_config_slug(self, info):
             return self.TargetingConfig(self.targeting_config_slug).value
         return self.targeting_config_slug
 
+    def resolve_targeting_configs(self, info):
```

Comment:
Also calling this method `resolve_targeting_configs` doesn't make sense on this object since a single experiment can only have a single targeting config, it should be singular.